content/webapp/pages/exhibition-guide.tsx

```diff
@@ -332,6 +387,13 @@ const ExhibitionLinks: FC<ExhibitionLinksProps> = ({ stops, pathname }) => {
         including descriptions of the objects."
           color="newPaletteSalmon"
           icon={audioDescribed}
+          onClick={event => {
+            event.stopPropagation();
+            cookieHandler(
+              'WC_userPreferenceGuideType',
+              'audio-with-descriptions'
+            );
+          }}
```

Comment:
Is there any case in which the `onclick` attribute of a TypeOption wouldn't set a cookie? And if so, could we pass in the guide ID and set the cookie inside the TypeOption component instead?

Reply:
Good point